runtime/src/accounts_background_service.rs

```diff
@@ -389,14 +388,6 @@ impl SnapshotRequestHandler {
             snapshot_root_bank.hash(),
         );
 
-        // Cleanup outdated snapshots
-        let mut purge_old_snapshots_time = Measure::start("purge_old_snapshots_time");
```

Comment:
this is no longer necessary because either we are not generating snapshots, and AHV purges after hash verification. OR we ARE generating snapshots and SPS purges it after archiving.